projects/contracts/src/lib/types.ts

```diff
@@ -26,15 +26,15 @@ export interface Clip extends HasId {
   type: MediaType;
 
   tags?: string[];  // All normal Media-Types can use that to be "tagged"
-                    // the Meta Type will use that to trigger all clips of that tagId
+  // the Meta Type will use that to trigger all clips of that tagId
```

Comment:
this probably wont be the last place where the comment needs to be there, is there a rule to disable this too? or should the comment be above the property?

Reply:
I don't know, the comment was there when I opened the file, so I just followed the eslint direction to refactor it as it was complaining about extra tabs. 

The rule for it is: `@typescript-eslint/indent`